src/components/tools/table-tool/data-table.tsx

```diff
@@ -289,15 +300,7 @@ export default class DataTableComponent extends React.Component<IProps, IState>
         if (params.data.id === LOCAL_ROW_ID) {
           return attrID ? this.localRow[attrID] : undefined;
         }
-        let value = dataSet && attrID ? dataSet.getValue(caseID, attrID) : undefined;
-        // valueGetter includes in-flight changes
-        this.localChanges.forEach((change) => {
-          if ((change.__id__ === caseID) && (attrID != null)) {
-            if (change[attrID] != null) {
-              value = change[attrID] as IValueType;
-            }
-          }
-        });
+        const value = dataSet && attrID ? dataSet.getValue(caseID, attrID) : undefined;
```

Comment:
Comment about this change....was overwriting expr values but it was there to get new values before they come back from Firebase + could blow up